From 5ac0593db9a8ef46f46f8b71440c71494d026644 Mon Sep 17 00:00:00 2001
From: Neal Liu <neal_liu@aspeedtech.com>
Date: Fri, 12 Sep 2025 14:53:11 +0800
Subject: [PATCH 27/32] include: evb_ast2700: set verify option to no
 temporarily

Set verify option to no temporarily.

Signed-off-by: Neal Liu <neal_liu@aspeedtech.com>
Change-Id: I992eccceefe41d4a924a8bb43e0f73b27d386c51
(cherry picked from commit 91b32ca122a023eb69366098d4f6f6cdcea1a45b)
Signed-off-by: Vasanthakumar K <vasanthakumark@ami.com>
---
 include/configs/evb_ast2700.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/configs/evb_ast2700.h b/include/configs/evb_ast2700.h
index 202ff58cf7..764daf4544 100644
--- a/include/configs/evb_ast2700.h
+++ b/include/configs/evb_ast2700.h
@@ -29,6 +29,6 @@
 	"ufsboota=setenv bootpart 2; setenv rootfs rofs-a; run setufsargs; ext4load scsi 0:${bootpart} ${loadaddr} fitImage && bootm ${loadaddr}; echo Error loading kernel FIT image\0"	\
 	"ufsbootb=setenv bootpart 3; setenv rootfs rofs-b; run setufsargs; ext4load scsi 0:${bootpart} ${loadaddr} fitImage && bootm ${loadaddr}; echo Error loading kernel FIT image\0"	\
 	"bootufs=if test \"${bootside}\" = \"b\"; then run ufsbootb; run ufsboota; else run ufsboota; run ufsbootb; fi\0"	\
-	"verify=yes\0"	\
+	"verify=no\0"	\
 	""
 #endif	/* __CONFIG_H */
-- 
2.34.1

